QUARKUS-2739.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-2739 - Add support for JDBCStore transaction logs
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-2739
+
+Add support for `JDBC journal transaction logs`.
+
+## Scope of the testing
+
+A database transaction log should records all transactions and the database modifications made by each transaction. 
+If there is any fatal system failure, this log should be enough to recover your database back to a consistent state.
+
+Ensure that Narayana extension deals well with JDBC datasource transaction log and all the pending transactions (due to a crash 
+on Quarkus app) are completed when the application recover the initial state. 
+
+Tests should verify:
+- All supported datasources 
+- JVM and Native mode
+- Openshift integration (We know that Openshift will require some operator or persistent volumn in a none ephimeral node in order to test this feature, but this operator doesn´t exist at this moment. So the coverage is going to assume that all K8s / Openshift configuration is stable and mature, but maybe sometimes we could experiment some flaky test due to the ephimeral persistence unit that is under the hood)
+
+### Existing tests
+
+On Quarkus test suite there are already a Narayana module that could be reused in order to verify `JDBCStore transaction logs`.
+
+### Impact on resources
+
+Required additional time for the test execution should be around 10 minute for native platform and 5 min for JVM. Native additional time is due to the native compilation, that is going to be done in a completly new scenario decouple of the current scenarios. Is important to us have a clean scenario because we are going to monitor internal `JBOSSTSTXTABLE` db table where all the journal is stored.
```

Comment:
Is important to us have a clean scenario because we are going to monitor internal `JBOSSTSTXTABLE` db table where all the journal is stored.
 =>
It is important to have a new scenario because the test is going to monitor the `JBOSSTSTXTABLE` table where all the journal entries are stored.